yarn.lock

```diff
@@ -3670,81 +3670,69 @@
     "@webassemblyjs/wast-parser" "1.8.5"
     "@xtuc/long" "4.2.2"
 
-"@webb-tools/api@0.1.2-18", "@webb-tools/api@^0.1.2-22", "@webb-tools/api@^0.1.2-34", "@webb-tools/api@^0.1.2-37":
-  version "0.1.2-37"
-  resolved "https://registry.yarnpkg.com/@webb-tools/api/-/api-0.1.2-37.tgz#aff4113c6e80d79932f7dd5d2a3bcc729665abe6"
-  integrity sha512-1mu8eKnz7nq8GqRJrM28tfJKGtjtJJ2Ga9fEeNazPSuBgharwRczc1rUPComS6cFl2mbzlECLt9UkXijpUWJCA==
+"@webb-tools/api@0.1.2-18", "@webb-tools/api@^0.1.2-22", "@webb-tools/api@^0.1.2-41", "@webb-tools/api@file:../webb.js/packages/api/build":
+  version "0.1.2-41"
```

Comment:
@AhmedKorim maybe you need to run `yarn` again? All versions points to 41 even though from above it looks like we should be on 42. Also, please remove resolutions when you are submitting updates to the lock file.

Reply:
this is from sub-packages, the version here is obsoleted by yarn resolution